### ISSUE TITLE
Death by Asterix

### ISSUE THREAD

**Issue by aethiopica** [rasendubi/uniorg#34]
Hi, just reporting a small bug. You can reproduce it if you have a standalone asterix on a line, anywhere in a file. Other than that, your library worked great on my 8MB of org-mode files.

I was using it from Deno, so that's why the stack may look a little odd.

```
Error: parseList: cannot find item. contentsBegin: 0, structure: []
    at Parser.parseList (https://cdn.skypack.dev/-/uniorg-parse@v0.5.0-0W3ExrYwyZt2hj4fFbYr/dist=es2019,mode=imports/optimized/uniorg-parse.js:2177:13)
    at Parser.parseElement (https://cdn.skypack.dev/-/uniorg-parse@v0.5.0-0W3ExrYwyZt2hj4fFbYr/dist=es2019,mode=imports/optimized/uniorg-parse.js:1598:19)
    at Parser.parseElements (https://cdn.skypack.dev/-/uniorg-parse@v0.5.0-0W3ExrYwyZt2hj4fFbYr/dist=es2019,mode=imports/optimized/uniorg-parse.js:1445:28)
    at Parser.parse (https://cdn.skypack.dev/-/uniorg-parse@v0.5.0-0W3ExrYwyZt2hj4fFbYr/dist=es2019,mode=imports/optimized/uniorg-parse.js:1432:27)
    at parse (https://cdn.skypack.dev/-/uniorg-parse@v0.5.0-0W3ExrYwyZt2hj4fFbYr/dist=es2019,mode=imports/optimized/uniorg-parse.js:1422:47)
    at parser (https://cdn.skypack.dev/-/uniorg-parse@v0.5.0-0W3ExrYwyZt2hj4fFbYr/dist=es2019,mode=imports/optimized/uniorg-parse.js:2644:34)
    at Function.parse (https://cdn.skypack.dev/-/unified@v10.1.2-xtnFuapI0dFO1gIeDZpf/dist=es2019,mode=imports/optimized/unified.js:154:12)
    at executor (https://cdn.skypack.dev/-/unified@v10.1.2-xtnFuapI0dFO1gIeDZpf/dist=es2019,mode=imports/optimized/unified.js:214:31)
    at new Promise (<anonymous>)
    at Function.process (https://cdn.skypack.dev/-/unified@v10.1.2-xtnFuapI0dFO1gIeDZpf/dist=es2019,mode=imports/optimized/unified.js:209:14)
```

**Comment by rasendubi** [comment]
Oh, wow. Thanks for reporting this—I'll take a look soon

**Comment by rasendubi** [comment]
fixed in 0.5.1
